Core/GDCore/Extensions/Builtin/CameraExtension.cpp

```diff
@@ -590,6 +590,19 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsCameraExtension(
       .SetDefaultValue("\"\"")
       .AddParameter("expression", _("New default Z order"));
 
+  extension
+      .AddExpressionAndCondition("number",
+                                 "LayerHighestZOrder",
+                                 _("Highest Z order of a layer"),
```

Comment:
The full name must be as concise as possible because it's used in a list.
```suggestion
                                 _("Layer highest Z order"),
```